0-bootstrap/main.tf

```diff
@@ -52,6 +52,35 @@ module "seed_bootstrap" {
   group_billing_admins    = var.group_billing_admins
   default_region          = var.default_region
   sa_enable_impersonation = true
+
+  activate_apis = [
+    "serviceusage.googleapis.com",
+    "servicenetworking.googleapis.com",
+    "compute.googleapis.com",
+    "logging.googleapis.com",
+    "bigquery.googleapis.com",
+    "cloudresourcemanager.googleapis.com",
+    "cloudbilling.googleapis.com",
+    "iam.googleapis.com",
+    "admin.googleapis.com",
+    "appengine.googleapis.com",
+    "storage-api.googleapis.com",
+    "monitoring.googleapis.com",
+    "accesscontextmanager.googleapis.com"
+  ]
+
+  sa_org_iam_permissions = [
+    "roles/accesscontextmanager.policyAdmin",
```

Comment:
@rjerrems this role and enabling the "accesscontextmanager.googleapis.com" were missing from the PR.
They are necessary for the VPC service Control configuration.

Reply:
Ok sounds good to me